package.json

```diff
@@ -39,8 +39,5 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  },
-  "devDependencies": {
```

Comment:
We need this!

Reply:
what do u mean? i dont think i removed anything other than u suggested XD